README.md

```diff
@@ -16,72 +16,77 @@ You can simply install it via opam in your current switch by running:
+[Dune's documentation](https://dune.readthedocs.io/en/latest/dune-files.html#vendored-dirs-since-1-11))
+so that Dune will only build the artifacts you need from that folder.
 
 From that point you should be able to run `dune build` and `dune runtest` as you normally would and
 build your entire project from scratch!
 
-### opam monorepo lock
+### `opam-monorepo lock`
 
 It's important to note that `opam monorepo lock` will only succeed if all of your non-virtual and
-non "base" dependencies (e.g. `ocaml` or `dune`) build with dune (i.e. directly depend on the `dune`
+non "base" dependencies (e.g., `ocaml` or `dune`) build with Dune (i.e., directly depend on the `dune`
 or `jbuilder` packages).
-If that's not the case the solver will report which packages don't build with dune.
+If that's not the case, the solver will report which packages don't build with dune.
 
-We maintain a [separate opam repository](https://github.com/dune-universe/opam-overlays) with dune
-ports of commonly used opam packages. If you have non-dune dependencies it is advised that you add
+We maintain a [separate opam repository](https://github.com/dune-universe/opam-overlays) with Dune
+ports of commonly used opam packages. If you have non-Dune dependencies, we advise adding
 this repository before running `opam monorepo lock`. You can do so by running the following command:
 ```
 opam repository add dune-universe git+https://github.com/dune-universe/opam-overlays.git
 ```
-Note that if it is not setup, the plugin will warn you.
+Note that if it's not setup, the plugin will warn you.
 
 The `lock` command takes your global and switch's opam configurations into account, meaning any
 opam repository or pins you set up will be picked up by the solver when resolving the full set of
 your project's dependencies.
 
-The generated lockfile is meant to be compatible with `opam` in such a way that running `opam
-install . --locked` should give you the same versions you would using `opam monorepo pull` in
-a reproducible way (i.e. independently of any change that might have happened on the upstream
-opam-repository) thanks to the `pin-depends`.
-You can use that property to your advantage by allowing one to choose between a "monorepo" or
+The generated lock file is meant to be compatible with `opam` in such a way that running 
+
+```
+opam install . --locked
+``` 
+
+should give you the same versions you would using `opam monorepo pull` in
+a reproducible way (i.e., independently of any change that might have happened on the upstream
+`opam-repository`) thanks to the `pin-depends`.
+You can use that property to your advantage by allowing one to choose between a "monorepo" or a
 regular opam workflow depending on the situation.
 
 You can also exclude packages from the set of packages to
-be vendored by `opam-monorepo`. To do so you can specify an additional field in
-your Opam file:
+be vendored by `opam-monorepo`. To do so, specify an additional field in
+your opam file:
```

Comment:
Is there any particular reason why Dune and Git are capitalized but not opam?

Reply:
Yes. Opam under the rules of English language _should be capitalised_, as it's a proper noun, but Thomas feels strongly about its official name being formatted in lower case. Dune and Git are also proper nouns, so should be capitalised when not referring to the command like `dune` and `git` . I've already spoken with the maintainers of Dune, and they agree with the Dune and `dune` formatting. As for Git, it's capitalised by its maintainers and across the web.